src/main/java/JavaQuestionTwo/home.java

```diff
@@ -0,0 +1,84 @@
+package JavaQuestionTwo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.util.*;
+public class home {
+	Scanner sc = new Scanner(System.in);
+	public static void main(String[] args) {
+
+		home homeObj = new home();
+		homeObj.createFile();
+		homeObj.takeUserInput();
```

Comment:
either make them static functions, and call them directly, or move them to a separate service call and keep this class just for the entrypoint.